### ISSUE TITLE
to and from revision for material

### ISSUE THREAD

**Issue by studiotomi** [TWChennai/gocd-git-path-material-plugin#41]
Is it possible to get the to and from revisions for a material when using this plugin? When I take a look at the envvar's, I only see the revisions for the materials using the standard set up from gocd.

I'd like to see the triggering revisions if possible:

    [go] setting environment variable 'GO_REVISION_mat_name' to value 'abc123'
    [go] setting environment variable 'GO_TO_REVISION_mat_name' to value 'abc123'
    [go] setting environment variable 'GO_FROM_REVISION_mat_name' to value '123abc'

**Comment by chadlwilson** [comment]
The environment variables are all set by the main GoCD agent runner and if I recall correctly, I don't think SCM plugins like this one have control over these - one can think of them as something set up by the "sandbox" the GoCD agent ensures for each build where it merges env vars from Gocd + Pipeline Environment + Pipeline + Stage + Job.

The `TO` revisions should be in `GO_SCM_mat_name_LABEL` vars, but not sure there is anywhere for the previous/`FROM` revision.

Are you saying though, that the plugin lacks env vars that are there when using the built-in Git material?

**Comment by studiotomi** [comment]
Yes, materials that do not use this plugin have the revision envvars, and materials that use this plugin are missing the envvars.
